packages/cli/templates/cloudflare/tsconfig.json

```diff
@@ -26,6 +26,7 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "baseUrl": ".",
+    "customConditions": ["webstudio"],
```

Comment:
will not work when we make it public

Reply:
We have it in others, is it in internal?

Reply:
Moved to internal, and saas-template